examples/chip-tool/commands/pairing/Commands.h

```diff
@@ -177,7 +177,7 @@ class PairAlreadyDiscovered : public PairingCommand
 {
 public:
     PairAlreadyDiscovered(CredentialIssuerCommands * credsIssuerConfig) :
-        PairingCommand("already-discovered", PairingMode::AlreadyDiscovered, PairingNetworkType::Ethernet, credsIssuerConfig)
+        PairingCommand("already-discovered", PairingMode::AlreadyDiscovered, PairingNetworkType::Network, credsIssuerConfig)
```

Comment:
The other option is to call this PairingNetworkType::NoNetworkConfigNeeded, just to be super-explicit.

Reply:
Or use the existing PairingNetworkType::None, if this has behavior identical to that one...